spark/src/test/scala/org/apache/spark/sql/delta/DeltaVacuumSuite.scala

```diff
@@ -798,8 +807,10 @@ class DeltaVacuumSuite
     }
   }
 
+  // TODO: There is somewhere in the code calling CanonicalPathFunction with an unescaped path
+  //  string. To be fixed.
```

Comment:
As in you will test this with special characters later? I thought you meant you were trying to check in a test broken by this change. 😅 

This is unrelated to this PR's actual code change, right?

Reply:
> you will test this with special characters later
Yes, now this test will fail if we test special characters.